apps/components-e2e/src/components/quick-filter/quick-filter.e2e.ts

```diff
@@ -343,10 +343,10 @@ test('should work with async data and handle distincts correctly', async (testCo
     // Expect Vienna and Graz to still be in the list.
     .expect(options.nth(0).textContent)
     // textContent is duplicated because of the highlight within the option
-    .eql('ViennaVienna')
+    .eql(' ViennaVienna')
```

Comment:
Why is there a space in there now? I'm confused.

Reply:
I get this error otherwise:

<img width="789" alt="Screenshot 2021-02-01 at 10 36 30" src="https://user-images.githubusercontent.com/57528898/106440661-61d37300-6479-11eb-94a1-906d21c85011.png">

Reply:
I'm almost 100% certain this can be tracked back to this change:
https://github.com/dynatrace-oss/barista/pull/1919/files#diff-e38531c013cd8f799b6f772d2d6f8dcddb0a621551c466d38b3eebbee33b76a5L86-R111

Reply:
Solved. TY!